iree/hal/vulkan/vulkan_device.cc

```diff
@@ -227,6 +224,7 @@ StatusOr<ref_ptr<VulkanDevice>> VulkanDevice::Create(
     VkPhysicalDevice physical_device,
     const ExtensibilitySpec& extensibility_spec,
     const ref_ptr<DynamicSymbols>& syms,
+    bool force_timeline_semaphore_emulation,
```

Comment:
move after extensibility_spec to match the order used elsewhere and keep syms at the end